cs251tk/toolkit/__main__.py

```diff
@@ -139,18 +139,30 @@ def main():
     if assignments:
         available_specs = set(assignments)
 
+        if ci:
+            ignored_specs = open('.cs251tkignore').read().split('\n')
+            logging.debug("Ignored specs: {}".format(ignored_specs))
+
+            for spec_to_use in assignments:
+                if spec_to_use in ignored_specs:
+                    available_specs.remove(spec_to_use)
+                    continue
```

Comment:
Something something build a set out of ignored_specs, then do `available_specs = available_specs.difference(ignored_specs)`?